pkg/startup/v7/data/worker/etc/origin/node/node-config.yaml

```diff
@@ -45,6 +45,8 @@ kubeletArguments:
   - {{ print "node-role.kubernetes.io/" .Role "=true" | quote }}
   pod-manifest-path:
   - /etc/origin/node/pods
+  read-only-port:
```

Comment:
This enabled a local unauthenticated kubelet port from which `/metrics` can be scraped. We need to discuss and fully understand the risks of doing this.